ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -166,7 +166,8 @@ private fun SelectPackageButton(
 private val PaywallViewState.Loaded.selectedLocalization: ProcessedLocalizedConfiguration
     get() = selectedPackage.localization
 
-@Preview(showBackground = true)
+@Preview(showBackground = true, locale = "en-rUS")
+@Preview(showBackground = true, locale = "es-rES")
```

Comment:
This is nice 👍🏻 

What's the `r` supposed to be for?

Reply:
It's for `region`. From the docs, that's the way if you want to specify region. We could also just specify the language, but this seemed more realistic

Reply:
Sorry, didn't post the docs link: https://developer.android.com/jetpack/compose/tooling/previews#preview-locale